shells/pipes-shell/source/pipe.js

```diff
@@ -64,8 +63,7 @@ const populateDispatcher = (dispatcher, composerFactory, storage, context, env)
     // TODO: eventually this should replace `spawn`. currently adding a parallel
     // API call, to not affect existing demos.
     runarc: async (msg, tid, bus) => {
```

Comment:
Curious -- do we still need `tid`, is it eventually going to go away?

Reply:
We are not using it for the newer work (i.e. the prototype etc), in fact I added a deprecated warning in the Java side to verify this mechanism isn't used any more.
But Scott asked me not to tear anything out of pipes-shell, to keep the old demos functional.